ekaspersen/catch-the-fox
# Integrate Prettier with Tailwind Plugin for Consistent Code Formatting

## Overview

This PR sets up Prettier along with the Tailwind plugin to enforce consistent code formatting throughout the project.

## Changes

- Installed Prettier as a dev dependency.
- Added Tailwind plugin for Prettier.
- Created a `.prettierrc` configuration file with tailored settings for our project.
- Modified `package.json` to include formatting scripts.

## Benefits

- **Consistency**: Ensures that all code follows a uniform style, making it easier to read and maintain.
- **Efficiency**: Automates the code formatting process, saving developers time during code reviews and edits.
- **Collaboration**: Makes it easier for multiple developers to work on the same project by resolving style disagreements automatically.

## How to Test

1. Pull the changes from this PR.
2. Install new dependencies with  `npm install`.
3. Run `npx prettier . --write` to format the codebase.
4. Verify that the code is consistently formatted and adheres to the project's style guide.
